src/app/core/cache/models/normalized-bitstream-format.model.ts

```diff
@@ -0,0 +1,35 @@
+import { autoserialize, autoserializeAs, inheritSerialization } from 'cerialize';
+import { BitstreamFormat } from '../../shared/bitstream-format.model';
+
+import { mapsTo } from '../builders/build-decorators';
+import { IDToUUIDSerializer } from '../it-to-uuid-serializer';
+import { NormalizedObject } from './normalized-object.model';
+
+@mapsTo(BitstreamFormat)
+@inheritSerialization(NormalizedObject)
+export class NormalizedBitstreamFormat extends NormalizedObject {
```

Comment:
This Class should have comments to describe it & its properties. What is "normalized" about this BitstreamFormat object?